src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,102 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final double CAPACITY_MULTIPLIER = 1.5;
+    private static final int DEFAULT_CAPACITY = 10;
+
```

Comment:
No need in empty line

Reply:
actually, static fields are usually separated from the usual ones 😉